src/main/java/com/shnupbups/gentlepicking/GentlePicking.java

```diff
@@ -2,39 +2,25 @@
 
 import net.fabricmc.api.ModInitializer;
 import net.fabricmc.fabric.api.event.player.UseBlockCallback;
-import net.fabricmc.fabric.api.tag.TagFactory;
-import net.fabricmc.fabric.api.tag.TagRegistry;
-
 import net.minecraft.block.Block;
-import net.minecraft.block.BlockState;
-import net.minecraft.block.Blocks;
-import net.minecraft.item.BlockItem;
-import net.minecraft.item.ItemStack;
-import net.minecraft.loot.context.LootContext;
-import net.minecraft.loot.context.LootContextParameters;
-import net.minecraft.server.world.ServerWorld;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.tag.TagKey;
 import net.minecraft.sound.SoundCategory;
 import net.minecraft.sound.SoundEvents;
-import net.minecraft.tag.BlockTags;
-import net.minecraft.tag.ItemTags;
-import net.minecraft.tag.Tag;
 import net.minecraft.util.ActionResult;
-import net.minecraft.util.Hand;
 import net.minecraft.util.Identifier;
-import net.minecraft.util.math.BlockPos;
-import net.minecraft.util.math.Vec3d;
 
 public class GentlePicking implements ModInitializer {
 	public static final String MOD_ID = "gentlepicking";
 
-	public static final Tag<Block> PICKABLE = TagFactory.BLOCK.create(id("pickable"));
+	public static final TagKey<Block> PICKABLE = TagKey.of(Registries.BLOCK.getKey(), id("pickable"));
 
 	@Override
 	public void onInitialize() {
 		UseBlockCallback.EVENT.register((player, world, hand, hitResult) -> {
-			BlockPos pos = hitResult.getBlockPos();
-			BlockState state = world.getBlockState(pos);
-			if(state.isIn(PICKABLE) && player.canModifyBlocks() && !player.shouldCancelInteraction() && world.canPlayerModifyAt(player, pos) && !player.isSpectator()) {
+			var pos = hitResult.getBlockPos();
+			var state = world.getBlockState(pos);
```

Comment:
I prefer not to use `var` in my projects unless it specifically helps readability